### ISSUE TITLE
Unable to delete local branch with slashes in the name

### ISSUE THREAD

**Issue by arimbun** [textmate/git.tmbundle#40]
If you are trying to delete a local branch called `feature/some-branch`, TextMate will try to look for `feature`, ignoring the bits that come after.

```
I didn't understand's git's response, perhaps you can make sense of it?
fatal: 'feature' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by sorbits** [comment]
Fixed by PR #42.

**Comment by arimbun** [comment]
@sorbits https://github.com/textmate/git.tmbundle/pull/44 fixes this issue.
